archive/tartest/tar.go

```diff
@@ -68,14 +68,14 @@ func TarFromWriterTo(wt WriterToTar) io.ReadCloser {
 
 // TarContext is used to create tar records
 type TarContext struct {
-	UID int
-	GID int
 
 	// ModTime sets the modtimes for all files, if nil the current time
 	// is used for each file when it was written
 	ModTime *time.Time
 
 	Xattrs map[string]string
+	UID    int
+	GID    int
```

Comment:
I think it would be better to put the user and group information on top of the other information of the file